### ISSUE TITLE
[enhancement] DeepKoopman regressor

### ISSUE THREAD

**Issue by Enderdead** [dynamicslab/pykoopman#7]
Hello, 
I took a quick look at your library and got surprised that no deepkoopman technique was implemented.
Do you plan to include it in the future ? ( Even if this may lead to include a deep-learning dependency to your project. )
 
I would be happy to try to submit this feature  if you interested to include it.

Best regards

**Comment by eurika-kaiser** [comment]
Hi!

Thanks for your interest in the pykoopman package!
The package architecture has been developed to allow for extensions such as deep learning via plugins. If you would like to pursue this (which would be very cool!), I suggest to create a separate GitHub project, in which you develop this and I’d be happy to include your extension in the README. Note that the pykoopman package is still under development and I can let you know once there is a stable version (~ 1 month latest).

**Comment by Enderdead** [comment]
Hi, 
Thank you for your answer. 
I'm looking forward to your final version and your stable API before starting my development. 
When I will be happy with my work, I will reopen another issue in this repo.
Good luck for the end of your development.
